bbb-graphql-actions/src/actions/pluginPersistEvent.ts

```
@@ -0,0 +1,33 @@
+  throwErrorIfInvalidInput(input,
+    [
+      {name: 'pluginName', type: 'string', required: true},
+      {name: 'eventName', type: 'string', required: true},
+      {name: 'payload', type: 'json', required: true},
+    ]
+  )
+
+  const eventName = `PluginPersistEventMsg`;
+
+  const routing = {
+    meetingId: sessionVariables['x-hasura-meetingid'] as String,
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)